src/irc/IrcServer.ts

```diff
@@ -43,13 +43,13 @@ export interface IrcServerConfig {
     dynamicChannels: {
```

Comment:
A lot of these changes are making some of these configs optional, because we actually do want most of them to be optional.